ompi/mca/pml/yalla/pml_yalla.c

```diff
@@ -716,8 +717,12 @@ int mca_pml_yalla_start(size_t count, ompi_request_t** requests)
                 }
             }
         } else {
+            mca_pml_yalla_recv_request_t *rreq;
+            rreq = (mca_pml_yalla_recv_request_t *)req;
```

Comment:
@yosefe 
Don't see it in https://github.com/open-mpi/ompi/wiki/CodingStyle
But can move rreq/sreq declarations to the top if preferred.